DittoToolsApp/DittoToolsApp.xcodeproj/project.pbxproj

```diff
@@ -747,30 +751,37 @@
 		};
 /* End XCConfigurationList section */
 
+/* Begin XCRemoteSwiftPackageReference section */
+		0E52DC072989DFA800ACE97A /* XCRemoteSwiftPackageReference "DittoSwiftTools" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/getditto/DittoSwiftTools";
```

Comment:
Xcode is so verbose, but this is another copy of the "use what's in the `main` branch" which prevents you from being able to test the "current package version" in this branch or even as you make changes locally.